Source/Particles/ElementaryProcess/QEDInternals/BreitWheelerEngineWrapper.cpp

```diff
@@ -6,14 +6,20 @@
  */
 #include "BreitWheelerEngineWrapper.H"
 
-#include "QedTableParserHelperFunctions.H"
-#include "BreitWheelerDummyTable.H"
+#include <utils/serialization.hpp>
```

Comment:
This include name might be a bit too generic. Not sure if it is too much of an issue though.

Reply:
I left this as an open point because I feel that it requires an in-depth discussion, since it concerns `#include` conventions in WarpX. 

An easy solution would be to write something like:
 `#include <qed/src/utils/serialization.hpp>`

Another option would be to rename all the source files in picsar so that they start with `Picsar`.

Reply:
@ax3l , do you have an opinion on that?